### PR TITLE
[Docs] Replace preview image with gif

### PR DESCRIPTION
**Pull request by huong-li-nguyen**
## Description
- Replace image with newly created gif by @Joseph-Perkins  ⭐ 

See: https://github.com/mckinsey/vizro/tree/docs/update-preview-img

## Screenshot

Screenshot 2024-10-03 at 15 16 56

## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

  - I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
  - I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorized to submit this contribution on behalf of the original creator(s) or their licensees.
  - I certify that the use of this contribution as authorized by the Apache 2.0 license does not violate the intellectual property rights of anyone else.
  - I have not referenced individuals, products or companies in any commits, directly or indirectly.
  - I have not added data or restricted code in any commits, directly or indirectly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@
 ---
 
 <p align="center">
-<img src="https://raw.githubusercontent.com/mckinsey/vizro/main/.github/images/example_screens.png" width="700"/>
+<img src="https://raw.githubusercontent.com/mckinsey/vizro/docs/update-preview-img/.github/images/vizro_spash_teaser.gif" width="600"/>
```
Comment (huong-li-nguyen) [inline README.md:33]: Change to main

File: vizro-core/README.md [hunk 2]
```diff
@@ -30,7 +30,7 @@
 ---
 
 <p align="center">
-<img src="https://raw.githubusercontent.com/mckinsey/vizro/main/.github/images/example_screens.png" width="700"/>
+<img src="https://raw.githubusercontent.com/mckinsey/vizro/docs/update-preview-img/.github/images/vizro_spash_teaser.gif" width="600"/>
```
Comment (huong-li-nguyen) [inline vizro-core/README.md:33]: Change to main

**Review by petar-qb — APPROVED** [review]
I love this gif 👍

**Review by antonymilne — APPROVED** [review]
Doesn't look quite so snazzy on a dark background but still very nice 😀 

image

**Comment by petar-qb** [comment]
Should we add it here https://huggingface.co/vizro?

**Pull request merged**